packages/mdl2-theme/src/Mdl2Customizations.ts

```diff
@@ -0,0 +1,13 @@
+import { Mdl2Theme } from './mdl2/Mdl2Theme';
+import { Mdl2Styles } from './mdl2/Mdl2Styles';
+import { ICustomizations } from 'office-ui-fabric-react';
+import { addVariants } from '@uifabric/variants';
+
+export const Mdl2Customizations: ICustomizations = {
```

Comment:
Since MDL2 is an acronym, we should consider making it uppercase in the various const/type names. 
```suggestion
export const MDL2Customizations: ICustomizations = {
```

Reply:
Done